### PR TITLE
Update Queue docs

### PR DESCRIPTION
**Pull request by fakela**

### CODE REVIEW

File: pages/14.Queue/docs.en.md [hunk 1]
```diff
@@ -11,13 +11,11 @@ taxonomy:
 
 -----------
 
+> **_INfO:_**  Mautic 3.x users who are implementing RabbitMQ or Beanstalkd need to configure the settings directly in their local configuration file. If you are using the legacy Mautic 2.x series the steps below remains the same.
```
Comment (RCheesley) [inline pages/14.Queue/docs.en.md:14]: We can use the pretty styling here for a notes flash - they are documented in this PR: https://github.com/mautic/mautic-documentation/pull/162

```suggestion
> **_INFO:_**  Mautic 3.x users who are implementing RabbitMQ or Beanstalkd need to configure the settings directly in their local configuration file. If you are using the legacy Mautic 2.x series the steps below remains the same. 
```

**Comment by fakela** [comment]
@RCheesley can you please tell me the syntax/configuration example that a user can use when trying to configure this in the local.php file.

also, i added info at the top to alert both old and new Mautic users of the change

**Review by RCheesley — CHANGES_REQUESTED** [review]

File: pages/14.Queue/docs.en.md (hunk 1 above)
Comment (RCheesley) [reply pages/14.Queue/docs.en.md:14]: Can we give an example of the syntax to use?

**Comment by RCheesley** [comment]
@fakela I would suggest pinging the Product Team to ask for guidance on the syntax to be used - @mohit-rocks @npracht @jos0405 can you provide an example so we can get that documented?

**Comment by RCheesley** [comment]
This PR needs to be updated with the latest changes, and have the configuration settings included. Please can you find someone to work on this @fakela ?

**Comment by fakela** [comment]
This PR settles this https://github.com/mautic/mautic-documentation/pull/189 I will proceed to close this

**Pull request closed without merging**